extensions/infobip/README.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Infobip
+description: Infobip is a global cloud communications platform that provides a wide range of communication and customer engagement solutions for businesses.
+---
+
+# Infobip
+
+Infobip is a global cloud communications platform that provides a wide range of communication and customer engagement solutions for businesses. The company offers a suite of services and tools designed to help organizations connect with their customers, clients, and partners through various communication channels, including SMS, email, voice, chat apps, and more.
+
+## Extension settings
+
+In order to set up this extension, you will minimaly need to provide:
+
+1. Your unique API [base URL](https://www.infobip.com/docs/essentials/base-url)
+2. An [API key](https://www.infobip.com/docs/essentials/api-authentication)
+
+When utilizing the Infobip extension for sending emails and text messages, you'll need a 'from email' address and a 'from phone number' respectively.
+
+## Actions
+
+### Send SMS
+
+Send a text message to a recipient.
+
+### Send email
+
+Sends a simple email to a recipient.
+
+### Send email with template
+
+Send an email, using a template, to a recipient. You need to create a [Broadcast email template](https://www.infobip.com/docs/email/templates) as that is the only type of template supported by Infobip's API. Templates can be created and managed via the Infobip web portal.
+
+When using this action, the from/sender details and subject as specified in the template will be used. However, you can specify a subject if you would like to overwrite the subject specified in the template.
+
+Within your template, you can harness placeholders using Infobip's `{{` syntax. This feature allows you to craft templates with both static and dynamic content. You can determine the dynamic content's value when configuring the action in Awell Studio. For instance, if your template includes two placeholders, `{{variable_1}}` and `{{variable_2}}`, you can provide the following data in the placeholders action field to set values for these placeholders:
```

Comment:
I am a bit lost with this comment. The template, including the placeholders/variables is built in Infobip - not in Awell.

Example of what a template can look like in Infobip:

```html
<img src="mylogo.png" alt="Logo" />
<h1>Welcome</h1>
<p>Hi {{name}},</p>
<p>You have a visit planned on {{visitDate}}.</p>
```

The "Send email with template" action allows you to send an email based on a template (by specifying the template ID" and allows you to pass values for the placeholders/variables in the Infobip template. This can be done via the Infobip API simply by providing a key-value pair JSON object. This is where the `placeholders` action field is for:

```json
{
  name: "Nick",
  visitDate: "2023-10-18"
}
```

We will render a JSON action field type and users can either pass static data for the placeholders or insert Awell data variables:

```json
{
  name: "<AWELL_DATA_VAR_1>",
  visitDate: "<AWELL_DATA_VAR_2>"
}
```

Reply:
Ah ok, I thought the action involved writing Infobip variable placeholders in the action config itself, it's just passing the content as an object, similar to what we do for sendgrid templates. The description confused me, my bad. I'll resolve this